roles/openshift_logging/templates/rolebinding.j2

```diff
@@ -1,12 +1,21 @@
 apiVersion: v1
-kind: RoleBinding
+kind: {% if cluster is defined and cluster %}Cluster{% endif %}RoleBinding
```

Comment:
I should rebase the branch on the latest logging where we have created a 'clusterrolebinding' template.  Sorry for being lax